### PR TITLE
Comment edit post

### PR DESCRIPTION
**Pull request by Rockyraven**
some css and signup update

### CODE REVIEW

File: src/pages/homepage/homepage.css [hunk 1]
```diff
@@ -21,7 +21,7 @@
 .create-post{
     height: 10rem;
     width: 45rem;
-    border: 1px solid gray;
+    /* border: 1px solid rgb(231, 228, 228); */
 }
```
Comment (Shubhamkr2610) [inline src/pages/homepage/homepage.css:25]: remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]

File: src/component/post/Posts.jsx [hunk 2]
```diff
@@ -91,10 +91,18 @@ export const Posts = ({
...
       : setIsTooltipVisible((prev) => "hidden");
   };
+
+  // useEffect(() => {
+  //   const post = posts?.find((post) => post._id === _id);
+  //   const liked = post.likes.likedBy.some((it) => it._id === user._id);
+  //   setIsLiked(liked);
+  //   console.log(post);
+  //   console.log(liked);
+  // }, [posts]);
```
Comment (Shubhamkr2610) [inline src/component/post/Posts.jsx:101]: remove unused code else if these code is for future use , mention in comment

File: src/redux/slices/authSlice.js [hunk 3]
```diff
@@ -19,21 +19,23 @@ export const login = createAsyncThunk("auth/login", async (userData) => {
...
+        username: username,
         firstName: firstName,
         lastName: lastName,
         userphoto: userphoto,
         password1: password1
       },
      
     );
+    console.log(res.data);
     return res.data;
```
Comment (Shubhamkr2610) [inline src/redux/slices/authSlice.js]: always remove console while pushing the code

File: src/redux/slices/postSlice.js [hunk 4]
```diff
@@ -20,6 +20,7 @@ export const createPost = createAsyncThunk("/post/create", async (data, thunkAPI
           },
         }
       );
+      console.log(res.data.posts);
```
Comment (Shubhamkr2610) [inline src/redux/slices/postSlice.js]: there also needs to remove the console

**Review by Shubhamkr2610 — APPROVED** [review]
Gave some suggestion . After making the suggested changes you can proceed for merge .
Appreciation : code is clean and easy to understand , good naming of state and variable

**Review by arjun-computer-geek — APPROVED** [review]
Everything is looking good but always try to write good commit messages

File: src/redux/slices/authSlice.js (hunk 3 above)
Comment (Rockyraven) [reply src/redux/slices/authSlice.js]: removed

**Comment by Rockyraven** [comment]
> Everything is looking good but always try to write good commit messages
I will remind for later

**Pull request merged**